interp/interp_test.go

```diff
@@ -3780,7 +3780,7 @@ func TestRunnerRun(t *testing.T) {
 			if err := r.Run(ctx, file); err != nil {
 				cb.WriteString(err.Error())
 			}
-			want := c.want
+			want := strings.ReplaceAll(c.want, "#JUSTDIR", tdir)
```

Comment:
rather than adding `#JUSTDIR` to the test logic, what I'd do is report the error based on what the original path was, not the absolute path. that's more helpful to the user in most cases, and given that the tests don't really use absolute paths in these cases, it also avoids adding extra test logic here.